lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -50,6 +50,20 @@ function dosomething_helpers_menu() {
 }
 
 /**
+ * Utility function to convert dates to ISO-8601 datetime.
+ */
+function dosomething_helpers_convert_date($timing) {
```

Comment:
You can probably change up the variable names in this function now that it's a generic helper function. The `$timing` and `$season` variables were very specific to extracting the `high_season` and `low_season` dates, but if this is now more generic the variables should be as well :)

Might also change name to `dosomething_helpers_convert_date_to_iso()` to make it obvious to what it's converting the date to OR, pass a default parameter that is the format, such as:

```
dosomething_helpers_convert_date($date, $format = 'iso-8601') {
 if ($format === 'iso-8601') {
  ... do yo thang...
 }
}
```